src/main/webapp/app/course/manage/course-group.component.ts

```diff
@@ -272,4 +277,35 @@ export class CourseGroupComponent implements OnInit, OnDestroy {
         this.rowClass = className;
         setTimeout(() => (this.rowClass = undefined));
     };
+    /**
```

Comment:
```suggestion

    /**
```
Missing empty line between methods.